cmd/kaniko-docker/main.go

```diff
@@ -119,6 +121,16 @@ func main() {
 			Usage:  "Cache timeout in hours. Defaults to two weeks.",
 			EnvVar: "PLUGIN_CACHE_TTL",
 		},
+		cli.StringFlag{
+			Name:   "digest-file",
+			Usage:  "Digest file Location",
+			EnvVar: "PLUGIN_DIGEST_FILE",
+		},
+		cli.StringFlag{
+			Name:   "output-file",
+			Usage:  "output file Location",
+			EnvVar: "PLUGIN_OUTPUT_FILE",
```

Comment:
Use PLUGIN_ARTIFACT_FILE instead of PLUGIN_OUTPUT_FILE.

Reply:
done